src/modules/counterItem.js

```diff
@@ -0,0 +1,10 @@
+export function countDisplayedShows(displayEpisode) {
+  const arr = Array.from(displayEpisode.querySelectorAll('.card'));
+  return {
+    count: arr.length,
+  };
+}
+
+export function countShows(arr) {
+  return arr.length;
+}
```

Comment:
- [ ] Since you were asked to to use ES6, Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because you learned how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
Thank you for reviewing my code .

- [ ] Completed ✔️